projection/src/main/scala/org/apache/pekko/projection/r2dbc/internal/R2dbcOffsetStore.scala

```diff
@@ -24,33 +24,35 @@ import scala.annotation.tailrec
 import scala.collection.immutable
 import scala.concurrent.ExecutionContext
 import scala.concurrent.Future
-
-import org.apache.pekko
-import pekko.Done
-import pekko.actor.typed.ActorSystem
-import pekko.actor.typed.scaladsl.LoggerOps
-import pekko.annotation.InternalApi
-import pekko.dispatch.ExecutionContexts
-import pekko.persistence.Persistence
-import pekko.persistence.query.DurableStateChange
-import pekko.persistence.query.Offset
-import pekko.persistence.query.TimestampOffset
-import pekko.persistence.query.UpdatedDurableState
-import pekko.persistence.query.typed.EventEnvelope
-import pekko.persistence.query.typed.scaladsl.EventTimestampQuery
-import pekko.persistence.r2dbc.internal.R2dbcExecutor
-import pekko.persistence.r2dbc.internal.Sql.Interpolation
-import pekko.persistence.typed.PersistenceId
-import pekko.projection.BySlicesSourceProvider
-import pekko.projection.MergeableOffset
-import pekko.projection.ProjectionId
-import pekko.projection.internal.ManagementState
-import pekko.projection.internal.OffsetSerialization
-import pekko.projection.internal.OffsetSerialization.MultipleOffsets
-import pekko.projection.internal.OffsetSerialization.SingleOffset
-import pekko.projection.r2dbc.R2dbcProjectionSettings
 import io.r2dbc.spi.Connection
 import io.r2dbc.spi.Statement
+import org.apache.pekko
+import org.apache.pekko.Done
```

Comment:
please revert these import changes - imports start with 'pekko.' and we import 'org.apache.pekko' to allow that